packaging/Vagrantfile

```diff
@@ -23,31 +23,42 @@ PACKAGER_BRANCH = ENV['PACKAGER_BRANCH']
 REPO = ENV['REPO']
 GITHUB_USERNAME = ENV['GITHUB_USERNAME']
 GITHUB_PASSWORD = ENV['GITHUB_PASSWORD']
+WORKSPACE = ENV['WORKSPACE']
 
 CENTOS7_OFFICIAL_BOX_URL = 'http://cloud.centos.org/centos/7/vagrant/x86_64/images/CentOS-7.box'
+CENTOS7_AMI = "ami-8df8c1eb"
+CENTOS7_USER = "centos"
+
+AWS_USER_DATA = %(
+#cloud-config
+write_files:
+-   path: /etc/sudoers.d/999-vagrant-cloud-init-requiretty
+    permissions: 440
+    content: |
+        Defaults:centos !requiretty
+)
 
 Vagrant.configure('2') do |config|
-  config.vm.define "restservice" do |rest|
+  config.vm.define "builder" do |builder|
     #dummy box, will be overriden
```

Comment:
Is this still relevant?

Reply:
that's the vm that'll be used by build_rpm if you want it to run vagrant (it has a mode of running a vagrant box, and a mode of running mock locally)